src/odemis/gui/model/__init__.py

```diff
@@ -626,6 +626,39 @@ def __init__(self, main):
         self.spec_bck_cal.subscribe(self._on_spec_bck_cal)
         self.spec_cal.subscribe(self._on_spec_cal)
 
+        self.zpos = model.FloatContinuous(0, range=(0, 0), unit="m")
+        self.zpos.clip_on_range = True
+        self._updateZParams()
```

Comment:
No need to call `_updateZParams` now, as the `subscribe(..., init=True)` takes care of it.